frontend/src/app/features/work-packages/components/wp-card-view/wp-single-card/wp-single-card.component.ts

```diff
@@ -85,11 +95,22 @@ export class WorkPackageSingleCardComponent extends UntilDestroyedMixin implemen
 
   ngOnInit():void {
     // Update selection state
-    this.wpTableSelection.live$()
+    combineLatest([
+      this.wpTableSelection.live$(),
+      this.uiRouterGlobals.params$,
+    ])
       .pipe(
         this.untilDestroyed(),
+        map(() => {
+          if (this.selectedWhenOpen) {
+            return this.uiRouterGlobals.params.workPackageId === this.workPackage.id;
+          }
+
+          return this.wpTableSelection.isSelected(this.workPackage.id as string);
+        }),
       )
-      .subscribe(() => {
+      .subscribe((selected) => {
+        this.selected = selected;
```

Comment:
Kinda feels like this logic should be in the parent, and the children only receive `selected="true/false"`, tho the team-planner component is massive already so it also makes sens to have it here.

Reply:
Yes I agree. Right now the card distinguishes whether it is selected or not by checking the service. We should definitely make the card dumber, but that's not something I'd like to include in this PR, and instead extend it to the whole single card, as it has a lot of other template expressions as well.